tests/e2e/tests/logging_test.py

```diff
@@ -0,0 +1,30 @@
+"""E2E test for logging API.
+
+Currently DOES NOT work.
+"""
+import json
+import logging
+import os
+import pytest
+
+from google.appengine.api.logservice import logservice
+
+
+@pytest.fixture
+def request_id():
```

Comment:
I'll leave this fixture here.  Other tests I agree they're overkill, but this one is dependent per-request and if we ever upgrade our test harness to be able to test multiple requests this will be useful.